README.md

```diff
@@ -228,6 +228,30 @@ client.proxy_url('pod', 'podname', 5001, 'ns')
 ```
 
 
+#### Get the logs of a pod
+You can get the logs of a running pod:
+
+```ruby
+client.get_pod_log('pod-name', 'default')
+ => "Running...\nRunning...\nRunning...\n"
+```
+
+If that pod has more than one container, you must specify the container:
+
+```ruby
+client.get_pod_log('pod-name', 'default', container: 'ruby')
```

Comment:
Is this documented somewhere on k8s side that a container must be specified if there's more than one?